pyproject.toml

```diff
@@ -44,6 +47,7 @@ per-file-ignores = "gendiff/scripts/*.py: WPS421"
 [tool.pytest]
 norecursedirs = "__pycache__"
 addopts = "--strict-markers"
+exclude = ["tests/fixtures/", "tests/"]
```

Comment:
I'm not sure if it's right :) how pytest fill find test files if they are excluded?